### ISSUE TITLE
choose hats to avoid the Stroop effect?

### ISSUE THREAD

**Issue by josharian** [cursorless-dev/cursorless#1658]
This issue is going to be a hard one to address, so I expect it to stay open for a long time, probably indefinitely. But I thought I would raise it for fun and intellectual interest.

I have noticed that the token beneath the hat can interfere with my ability to pronounce the hat. For example, I have a harder time referring to the token `bit` when the hat is on the first letter (which I pronounce `bat`). This is likely due to interference, some analog of the [Stroop effect](https://en.wikipedia.org/wiki/Stroop_effect). If we had a good understanding of where that interference occurs, we could penalize hats that would increase latency and/or error rate.

To actually address this, we would want to do some controlled experiments to convince ourselves that it is a real problem (not just something that Josh imagined from his phenomenological observations), characterize the problem more precisely, get access to talon's spoken form of the hats, and then do something about it. Unless there is a cognitive science graduate student who has an interest in voice coding, I suspect that step one will never happen.

**Comment by pokey** [comment]
I've definitely noticed it when the hat is on the first letter. Doesn't even need to be particularly similar to "bat". That specific case is a fairly easy one to address. 

But interesting idea to think about the more general phenomenon

**Comment by josharian** [comment]
Good point! Might indeed be worth adding a cursorless setting for a first-alpha-letter hat penalty, defaulting to 0.

**Comment by pokey** [comment]
Ooh yeah that's an interesting idea
